src/expression.d

```diff
@@ -1564,10 +1585,11 @@ extern (C++) bool functionParameters(Loc loc, Scope* sc, TypeFunction tf, Type t
             }
             else
             {
-                //                arg = arg->isLvalue() ? callCpCtor(sc, arg) : valueNoDtor(arg);
+                //arg = doCopyOrMove(sc, arg);
```

Comment:
The bad indent in comment had been introduced by ddmd conversion. Originally the the comment start `//` was at 0 column in C++ code.